src/assets/AssetsContractController.ts

```diff
@@ -67,6 +72,8 @@ export class AssetsContractController extends BaseController<
 
   private erc20Standard?: ERC20Standard;
 
+  private ethQuery?: any;
```

Comment:
Could we use ES6 private fields instead of TypeScript? e.g. `#ethQuery` instead of `private ethQuery`. We wanted to move all private fields to `#private` eventually.

Reply:
Also, we have an `EthQuery` type. We don't need to use `any`.